test/unit/pages/patientnew.test.js

```diff
@@ -23,6 +23,7 @@ describe('PatientNew', function () {
       console.error = sinon.spy();
       var props = {
         fetchingUser: false,
+        onInviteMember: sinon.stub(),
```

Comment:
should we have some test that assert we trigger this when data is donated and the member invited?